mypyc/ir/class_ir.py

```diff
@@ -152,6 +152,18 @@ def __init__(self, name: str, module_name: str, is_trait: bool = False,
         # None if separate compilation prevents this from working
         self.children: Optional[List[ClassIR]] = []
 
+        # Instance attributes that are initialized in the class body.
+        self.attrs_with_defaults: Set[str] = set()
+
+        # Attributes that are always initialized in __init__ or class body
```

Comment:
Explain where they should be set here too, I think. Or choose one of this and the other comment to put the explanation.